src/Microsoft.ML.Data/Transforms/NAFilter.cs

```diff
@@ -79,10 +79,11 @@ private static VersionInfo GetVersionInfo()
         private const string RegistrationName = "MissingValueFilter";
 
         /// <summary>
```

Comment:
>   /// <summary> [](start = 5, length = 16)

remove

Reply:
I have extra comment in addition to `<include>` tag that's why I cannot remove this summary tag otherwise extra comments won't be visible in intellisense.

---
In reply to: [207688846](https://github.com/dotnet/machinelearning/pull/646#discussion_r207688846) [](ancestors = 207688846)